modules/ROOT/pages/amazon/amazon-sqs-connector.adoc

```diff
@@ -1,366 +1,79 @@
-		},
-		"NumberId": {
-			"stringValue" : "230.000000000000000001",
-			"dataType" : "Number"
-		} as Object {
-			class : "org.mule.extension.sqs.api.model.MessageAttributeValue"
-		}
-	} as Object {
-		class: "java.util.HashMap"
-	}
-} as Object {
-	class: "org.mule.extension.sqs.api.model.Message"
-}
-----
-+
-. Drag an Amazon SQS connector send message operation into the flow, and double-click the connector to open its Properties Editor.
-. In Basic Settings on the right side of the field Extension configuration click the plus icon and add values for Access Key, Secret Key, and Queue Name.
-. Configure the remaining parameters of the connector.
-+
-image::amazon/amazon-sqs-send-message.png[Send Message Parameters]
-+
-[%header%autowidth.spread]
-|===
-|Field |Value
-|Display Name |Enter a name for the connector operation.
-|Connector Configuration |Select the global configuration you create.
-|Message |`#[payload]`
-|===
-+
-. Add a Logger to print the response in the Mule Console.
-+
-image::amazon/amazon-sqs-demo-logger.jpg[]
-+
-[%header%autowidth.spread]
-|===
-|Field |Value
-|Display Name |Enter a name for the logger.
-|Message |Sent Message: `#[payload]`
-|Level |INFO (Default)
-|===
-+
-. Add another Amazon SQS connector > Get approximate number of messages to get the count of the messages in the queue.
-+
-image::amazon/amazon-sqs-demo-get-message-count.jpg[]
-+
-[%header%autowidth.spread]
-|===
-|Field |Value
-|Display Name |Enter a name for the connector operation.
-|Connector Configuration |Select the global configuration you create.
-|===
-+
-. Add a Logger to print the number in the Mule Console.
-+
-image::amazon/amazon-sqs-demo-logger2.jpg[]
 
-=== Create a Flow to Receive a Message
+== Compatibility
 
-This completes the first part of the use case. Now create another flow to receive message and log them before deleting them from the queue.
+Amazon SQS connector v5.4.1 is compatible with:
 
-. Drag an Amazon SQS connector and configure it as an inbound endpoint:
-+
-image::amazon/amazon-sqs-demo-receive-messages.png[]
-+
-[%header%autowidth.spread]
-|===
-|Field |Value
-|Display Name |Enter a name for the connector operation.
-|Connector Configuration |Select the global configuration you create.
-|Number of Messages |10
-|===
-+
-The Message processor's Queue URL attribute takes precedence over the Global Element Properties Queue URL. If none of the attributes belonging to Global Element Properties, including Queue Name, Queue URL, and the Message Processor's Queue URL is provided, the connector throws an exception.
-+
-. Add a Logger to print the message in the Mule Console:
-+
 [%header%autowidth.spread]
 |===
-|Field |Value
-|Display Name |Enter a name of your choice.
-|Message |#[payload]
-|Level |INFO (Default)
+|Application/Service |Version
+|Mule Runtime |4.1.x
+|AWS SDK for Java |1.11.604
 |===
 
+== Audience
+* Starting user -- To create your Mule app,
```

Comment:
Read the Amazon SQS Connector documentation that meets the needs of the type of user that you are:

```suggestion
* Starting user
** To create your Mule app,
```